src/info/infoBackup.h

```diff
@@ -61,6 +62,25 @@ Functions
 ***********************************************************************************************************************************/
 unsigned int infoBackupCheckPg(
     const InfoBackup *this, unsigned int pgVersion, uint64_t pgSystemId, uint32_t pgCatalogVersion, uint32_t pgControlVersion);
+// Remove a backup from the current section
+void infoBackupDataDelete(const InfoBackup *this, const String *backupDeleteLabel);
+void infoBackupSave(
+    InfoBackup *this, const Storage *storage, const String *fileName, CipherType cipherType, const String *cipherPass);
+
+/***********************************************************************************************************************************
+infoBackupDataLabelList - get a list of current backup labels
+***********************************************************************************************************************************/
+typedef struct InfoBackupDataLabelListParam
+{
+    const String *filter;
+} InfoBackupDataLabelListParam;
+
+#define infoBackupDataLabelListP(this, ...)                                                                                        \
+    infoBackupDataLabelList(this, (InfoBackupDataLabelListParam){__VA_ARGS__})
+#define infoBackupDataLabelListNP(this)                                                                                            \
+    infoBackupDataLabelList(this, (InfoBackupDataLabelListParam){0})
+
+StringList *infoBackupDataLabelList(const InfoBackup *this, InfoBackupDataLabelListParam param);
 
```

Comment:
As we discussed, have P/NP variants for a function with one optional param seems to be overkill.

Reply:
removed - now pass "this" and "expression"